### PR TITLE
Enable native cosmetic filtering on beta channel

### PR DESCRIPTION
**Pull request by antonok-edm**
Followup to #76, which was merged a week ago. I haven't seen any regressions reported so far.

### CODE REVIEW

**Comment by kjozwiak** [comment]
I'll wait till either @moritzhaller or @bridiver reviews the code before merging 👍

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers.

File: seed/seed.json
Comment (iefremov) [inline seed/seed.json:27]: this is wrong version. it should be `<chrome>.<brave>`, see other entries @antonok-edm @kjozwiak

Comment (iefremov) [reply seed/seed.json:27]: @moritzhaller maybe we could add a version validation step?

Comment (antonok-edm) [reply seed/seed.json:27]: ah, so it's the major version of Chromium followed by all 3 version sections of Brave?

At a first glance it looked just like a full Chromium version, so that's what I used. Probably would be good to add this information to the README at the very least.

**Pull request merged**
